src/scripts/_scoreboard.ts

```diff
@@ -6,12 +6,14 @@ const computerResult = document.querySelector(
 	'.js-computer-score'
 ) as HTMLParagraphElement;
 
-export const showScoreboard = (computerScore: number, playerScore: number) => {
-	playerResult.textContent = `${playerScore}`;
-	computerResult.textContent = `${computerScore}`;
-	scoreboard.classList.add('show');
-};
+export const showScoreboard = (COMPUTER_SCORE: number, PLAYER_SCORE: number, MAX_ATTEMPTS: number, ATTEMPT_COUNTER: number) => {
```

Comment:
Jak przyjmujesz parametry, to one nie muszą być tak samo nazwane. Tutaj raczej poszedłbym w kierunek camelCase normalnie. Ważna jest kolejność ich przyjmowania. Mimo to, zastanowiłbym się czy nie lepiej przyjmować tutaj obiekt z danymi. 4 parametry funkcji to już sporo, dodaj jeszcze piąty jako jakąś flagę i wywołanie tej funkcji z przekazaniem wszystkich parametrów będzie ciężkie.